samples/python/agents/ag2/pyproject.toml

```
@@ -5,11 +5,8 @@ description = "MCP agent using A2A and AG2"
 readme = "README.md"
 requires-python = ">=3.12"
 dependencies = [
-    "ag2>=0.8.6",
-    "ag2[mcp, openai]>=0.8.6",
+    "ag2>=0.9.2",
+    "ag2[mcp, openai]>=0.9.2",
     "google-genai>=1.10.0",
-    "a2a-samples",
-]
-
-[tool.uv.sources]
-a2a-samples = { workspace = true }
\ No newline at end of file
+    "a2a-sdk>=0.2.6",  # it can be added back as a member after https://github.com/ag2ai/ag2/pull/1881
```

Should this comment be here?

---

Hi! Good point let me clean up the comment. The comment should be here but my phrasing isn't clear.
```suggestion
    "a2a-sdk>=0.2.6",  # After https://github.com/ag2ai/ag2/pull/1881, this pyproject can be added back in as a member of samples/python/pyproject.toml
```